src/frontend/components/Header.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+export default function Header () {
+  return (
+    <>
+      <div className='hero is-success'>
+        <div className='centered'>
+          <nav className='breadcrumb centered' aria-label='breadcrumbs'>
+            <ul>
+              <a href='/' className='brand-logo'>LOGO</a>
+              <li><a href='/'>PicoCHAT</a></li>
+              <li><a href='#/policy'>Rules</a></li>
+              <li><a href='#/about'>Om Oss</a></li>
```

Comment:
`Om Oss` -> `About`, It's an international project, prototype in english.

Reply:
Already changed